community/hazelcast-jet-enterprise/values.yaml

```diff
@@ -0,0 +1,285 @@
+  enabled: true
+  ## Hazelcast Jet Management Center image version
+  ## ref: https://hub.docker.com/r/hazelcast/hazelcast-jet-management-center/tags/
+  ##
+  image:
+    # repository is the Hazelcast Jet Management Center image name
+    repository: "hazelcast/hazelcast-jet-management-center"
+    # tag is the Hazelcast Jet Management Center image tag
+    tag: "3.0"
+    # pullPolicy is the Docker image pull policy
+    # It's recommended to change this to 'Always' if the image tag is 'latest'
+    # ref: http://kubernetes.io/docs/user-guide/images/#updating-images
+    #
+    pullPolicy: IfNotPresent
+    # pullSecrets is an array of docker-registry secret names
+    # Secrets must be manually created in the namespace.
+    # ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+    pullSecrets:
+    # - myRegistryKeySecretName
+
+  # javaOpts are additional JAVA_OPTS properties for Hazelcast Jet Management Center
+  javaOpts:
+
+  # licenseKey is the license key for Hazelcast Jet Management Center
+  # if not provided, it can be filled in the Management Center web interface
+  licenseKey:
+  # licenseKeySecretName is the name of the secret where the Hazelcast Jet Management Center License Key is stored (can be used instead of licenseKey)
+  licenseKeySecretName: 
+
+  # nodeSelector is an array of Hazelcast Jet Management Center Node labels for POD assignments
+  # ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector
+  nodeSelector: {}
+
+  # configurationFiles are Hazelcast Jet Client configuration files which will be used by Hazelcast Jet Management Center
+  configurationFiles:
+    hazelcastClientXml: |-
+      <?xml version="1.0" encoding="UTF-8"?>
+      <hazelcast-client xsi:schemaLocation="http://www.hazelcast.com/schema/client-config hazelcast-client-config-3.10.xsd"
+                        xmlns="http://www.hazelcast.com/schema/client-config"
+                        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+        <properties>
+          <property name="hazelcast.discovery.enabled">true</property>
+        </properties>
+        <network>
+          <discovery-strategies>
+            <discovery-strategy enabled="true" class="com.hazelcast.kubernetes.HazelcastKubernetesDiscoveryStrategy">
+              <properties>
+                <property name="service-name">${serviceName}</property>
+                <property name="namespace">${namespace}</property>
+              </properties>
+            </discovery-strategy>
+          </discovery-strategies>
+        </network>
+      </hazelcast-client>
+
+  # Configure resource requests and limits
+  # ref: http://kubernetes.io/docs/user-guide/compute-resources/
+  #
+  resources:
+    requests:
+      memory:
+      cpu:
+    limits:
+      memory:
+      cpu:
+
+  # Hazelcast Jet Management Center Service properties
+  service:
```

Comment:
Please add managementcenter.service.clusterIP to the values.yaml, values-metadata.yaml, and README.md.